packages/shared/src/components/modals/CompanionModal.tsx

```diff
@@ -0,0 +1,128 @@
+import React, { ReactElement, MouseEvent } from 'react';
+
+import { LazyModalCommonProps, Modal } from './common/Modal';
+import { Button } from '../buttons/Button';
+import UpvoteIcon from '../icons/Upvote';
+import CommentIcon from '../icons/Discuss';
+import LineIcon from '../icons/Line';
+import { IconSize } from '../Icon';
+
+type Props = LazyModalCommonProps & {
+  url: string;
+  onRequestClose: (event: MouseEvent) => void;
+  onReadArticleClick: () => void;
+  onActivateCompanion: () => void;
+};
+
+export default function CompanionModal({
+  onRequestClose,
+  onReadArticleClick,
+  onActivateCompanion,
+  url,
+  ...props
+}: Props): ReactElement {
+  return (
+    <Modal
+      {...props}
+      className="border-none"
+      kind={Modal.Kind.FlexibleCenter}
+      size={Modal.Size.XLarge}
+      onRequestClose={onRequestClose}
+    >
+      {/* eslint-disable-next-line jsx-a11y/media-has-caption */}
+      <video
+        className="tablet:hidden rounded-16 w-fit"
+        poster="https://daily-now-res.cloudinary.com/image/upload/s--IqqAH4Rt--/f_auto/v1689497816/mobile-image_zb923g"
+        loop
+        autoPlay
+      >
+        <source
+          src="https://daily-now-res.cloudinary.com/video/upload/s--XAM05C_P--/v1689497504/final_mobile_cm24ad.mp4"
+          type="video/mp4"
+        />
+      </video>
+
+      {/* eslint-disable-next-line jsx-a11y/media-has-caption */}
+      <video
+        className="hidden tablet:block rounded-16 w-fit"
+        poster="https://daily-now-res.cloudinary.com/image/upload/s--7w2Deabi--/f_auto/v1689497816/web-image_yomdsz"
+        loop
+        autoPlay
+      >
+        <source
+          src="https://daily-now-res.cloudinary.com/video/upload/s--2N7_JImI--/v1689497504/final_web_rkupko.mp4"
+          type="video/mp4"
+        />
+      </video>
```

Comment:
this is bad, but we need a function which dictates if we are on desktop or mobile outside of css media queries so we can handle this. Is there anything already in the codebase for this? 

If not, I can add something, unless someone has a different idea.

Reply:
We do have a `useMedia` hook, that's likely the one you want to use here.